DDCore/src/VolumeManager.cpp

```diff
@@ -268,6 +268,7 @@ namespace dd4hep {
             context->element    = e;
             if ( nodes.size() > 0 )  {
               ContextExtension* ext = new(_getExtension(context)) ContextExtension();
+	      context->flag=true ;
```

Comment:
For consistency, please use spaces for indentation, and drop the space before the semicolon.